### DISCUSSION TITLE
Build catalog for CMIP6

### DISCUSSION THREAD

**Discussion by lee1043** [ncar-xdev/ecgtools#174] [Q&A]
Hi, I was following [this](https://ecgtools.readthedocs.io/en/latest/how-to/build-cmip6-catalog.html) documentation for testing making catalogue of a CMIP6 database but getting error from the following line of the code:

```python
b = Builder(root_path="/glade/collections/cmip/CMIP6/CFMIP/", depth=3, njobs=-1)
```

Error:
```
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
Cell In[5], line 1
----> 1 b = Builder(root_path="/p/css03/esgf_publish/CMIP6/CMIP", depth=3, njobs=-1)

File ~/.conda/envs/intake-esm/lib/python3.11/site-packages/pydantic/_internal/_dataclasses.py:124, in complete_dataclass..__init__(__dataclass_self__, *args, **kwargs)
    122 __tracebackhide__ = True
    123 s = __dataclass_self__
--> 124 s.__pydantic_validator__.validate_python(ArgsKwargs(args, kwargs), self_instance=s)

ValidationError: 1 validation error for Builder
paths
  Field required [type=missing, input_value=ArgsKwargs((), {'root_pat...depth': 3, 'njobs': -1}), input_type=ArgsKwargs]
    For further information visit https://errors.pydantic.dev/2.0.3/v/missing
```

According to the API description for the [`Builder`](https://ecgtools.readthedocs.io/en/latest/reference/index.html#builder), it take `paths`, instead of `root_path`. Is this an error in documentation or did I miss something?

Thank you!

**Comment by jsta** [comment]
I believe you are correct. A commit (https://github.com/ncar-xdev/ecgtools/commit/a4ec580dc55297dfa86bf1ecf6baa29480219c6e) was pushed fixing this documentation.
